src/scheduling/cost_models/simulated_dfs.cc

```diff
@@ -0,0 +1,105 @@
+// The Firmament project
+// Copyright (c) 2015 Adam Gleave <arg58@cam.ac.uk>
+
+#include "scheduling/cost_models/simulated_dfs.h"
+
+#include <algorithm>
+
+#include "base/common.h"
+#include "../ext/spooky_hash/SpookyV2.h"
```

Comment:
This is a little suboptimal, as the ext directory isn't part of our standard include path (and not in the source tree either). Should we just add it to the include path in Makefile.config?